src/Shared/Testimonials.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Navigation, Pagination, Scrollbar, A11y } from "swiper/modules";
+import "swiper/css/navigation";
+import "swiper/css/pagination";
+import "swiper/css/scrollbar";
+import "swiper/css";
+import { EffectFade } from "swiper/modules";
+
+const Testimonials = ({ cardData }) => {
+  return (
+    <div className="mt-10 p-10 mb-10">
```

Comment:
This is the container for Testimonials with some margins and paddings makes sense , but this should only be for testimonial section.
You need to apply Page padding in body or in app.jsx but only on one place. 
Thus your whole application will have the same outlined padding 
And apply padding margin to each section accordingly (if necessary)